dev/io.openliberty.microprofile.telemetry.2.0.logging.internal.container_fat/publish/files/traceSourceServer.xml

```diff
@@ -23,7 +23,4 @@
     
    	<mpTelemetry source="trace"/>
 
-
-
-
-</server>
+</server>
```

Comment:
Please remove the changes for this file from the commit, as there are no significant changes made.